Added docs Inventory Guide. (#10239)
ansible-collections/community.general: docs/docsite/rst/iocage_inventory_guide_tags.rst

..
  Copyright (c) Ansible Project
  GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
  SPDX-License-Identifier: GPL-3.0-or-later

.. _ansible_collections.community.general.docsite.guide_iocage.iocage_inventory_guide.iocage_inventory_guide_tags:

Tags
----

Quoting `man iocage <https://man.freebsd.org/cgi/man.cgi?query=iocage>`_:

   PROPERTIES
   ...
   notes="any string"
   Custom notes for miscellaneous tagging.
   Default: none

>    Source: local

Reviewer: Shouldn't this be a code block (with language `text`)?

Reviewer: In that example you were quoting text (with regular text formatting); here it looks more like some preformatted text that should be printed verbatim. I don't know how RST quotes behave with multi-line text, whether you get line-breaks at the exact positions or whether it will end up in one line.

It might be better to have empty lines between the lines if you want to keep the quoting:
```suggestion
   PROPERTIES

   ...

   notes="any string"

   Custom notes for miscellaneous tagging.

   Default: none

   Source: local
```

Also note that in the `man` page, you have indents. A code block (with reduced indents) is probably the better choice to quote a man page's content.

Author: I'll add the code block.

We will use the format `notes="tag1=value1 tag2=value2 ..."`.

> Note: The iocage tags has nothing to do with the :ref:`tags`

Reviewer: ```suggestion
.. note::

    The iocage tags have nothing to do with the :ref:`tags`.
```

As root at the iocage host, set notes. For example,

.. code-block:: console

   shell> iocage set notes="vmm=iocage_02 project=foo" srv_1
   notes: none -> vmm=iocage_02 project=foo
   shell> iocage set notes="vmm=iocage_02 project=foo" srv_2
   notes: none -> vmm=iocage_02 project=foo
   shell> iocage set notes="vmm=iocage_02 project=bar" srv_3
   notes: none -> vmm=iocage_02 project=bar

Update the inventory configuration. Compose a dictionary *iocage_tags* and create groups. The option

> :ansopt:`community.general.iocage#inventory:get_properties` must be enabled:
> 
> .. code-block:: console
> 
>    shell> cat hosts/02_iocage.yml

Reviewer: ```suggestion
:ansopt:`community.general.iocage#inventory:get_properties` must be enabled.
For example, ``hosts/02_iocage.yml`` could look like:
```

.. code-block:: yaml

   plugin: community.general.iocage
   host: 10.1.0.73
   user: admin
   get_properties: true
   hooks_results:
     - /var/db/dhclient-hook.address.epair0b
   compose:
     ansible_host: (iocage_hooks.0 == '-') | ternary(iocage_ip4, iocage_hooks.0)
     iocage_tags: dict(iocage_properties.notes | split | map('split', '='))
   keyed_groups:
     - prefix: vmm
       key: iocage_tags.vmm
     - prefix: project
       key: iocage_tags.project

> Display tags and groups. Create a playbook:
> 
> .. code-block:: console
> 
>    shell> cat pb-test-groups.yml

Reviewer: ```suggestion
Display tags and groups. Create a playbook ``pb-test-groups.yml``:
```
